testing/testcafe/tests/overlays/pages/popupDrag/allowDragOutside.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Popup drag</title>
+
+    <link rel="stylesheet" type="text/css" href="../../../../../../artifacts/css/dx.light.css"/>
+
+    <script type="text/javascript" src="../../../../../../artifacts/js/jquery.min.js"></script>
+    <script type="text/javascript" src="../../../../../../artifacts/js/dx.all.debug.js"></script>
+</head>
+<body>
+    <div id="popup"></div>
+    <script>
+        $(function() {
+            $("#popup").dxPopup({
+                width: 100,
+                height: 100,
+                visible: true,
+                dragEnabled: true,
+                allowDragOutside: true,
+                animation: true
```

Comment:
why `true`?

Reply:
that's typo, should be `undefined` for sure